src/api/python/cvc5.pxd

```diff
@@ -27,6 +27,29 @@ cdef extern from "<string>" namespace "std":
         const wchar_t* data() except +
         size_t size() except +
 
+cdef extern from "<tuple>" namespace "std" nogil:
+    cdef cppclass tuple[T, U, S]:
+        ctypedef T first_type
+        ctypedef U second_type
+        ctypedef S third_type
+        T first
+        U second
+        S third
+        tuple() except +
+        tuple(tuple&) except +
+        tuple(T&, U&, S&) except +
+        bint operator==(tuple&, tuple&)
+        bint operator!=(tuple&, tuple&)
+        bint operator<(tuple&, tuple&)
+        bint operator>(tuple&, tuple&)
+        bint operator<=(tuple&, tuple&)
+        bint operator>=(tuple&, tuple&)
+
+cdef extern from "<tuple>" namespace "std":
+    uint32_t get0 "std::get<0>"(tuple[uint32_t,uint32_t,Term]) except +
+    uint32_t get1 "std::get<1>"(tuple[uint32_t,uint32_t,Term]) except +
+    Term get2 "std::get<2>"(tuple[uint32_t,uint32_t,Term]) except +
```

Comment:
I think we can properly declare them a templated functions (not 100% sure about the syntax though, see https://cython.readthedocs.io/en/latest/src/userguide/wrapping_CPlusPlus.html#templates)
```suggestion
    T get0[T,U.S] "std::get<0>"(tuple[T,U,S]) except +
    U get1[T,U.S] "std::get<1>"(tuple[T,U,S]) except +
    S get2[T,U.S] "std::get<2>"(tuple[T,U,S]) except +
```

Reply:
Discussed offline. This does not work, and we don't know how to make it work...